doc/authentication.md

```diff
@@ -39,7 +89,34 @@ BoxAPIConnection api = new BoxAPIConnection("YOUR-CLIENT-ID",
     "YOUR-CLIENT-SECRET", "YOUR-AUTH-CODE");
 ```
 
-### Manual Authentication
+### Box View Authentication with App Token
+//start with the normal BoxTransactionAPIConnection
+
+Allows applications that do not own content stored in Box(e.g. app-owned content) to be able to use Box as a service
+provider rather than a content store. This is currently mostly used for previewing items. For scopes you can choose 
+between "item_preview", "item_upload", or "item_delete". See the 
+[Getting Started with the New Box View](https://developer.box.com/docs/getting-started-with-new-box-view) for detailed
+instruction.
+
+```java
+BoxTransactionalAPIConnection api = new BoxTransactionalAPIConnection("YOUR-ACCESS-TOKEN");
+```
+
+You can also request a specific scope of the transaction token by passing in: "item_preview", "item_upload", or "item_delete". 
+
+```java
+BoxAPIConnection api = BoxTransactionalAPIConnection.getTransactionConnection("YOUR-ACCESS-TOKEN", "item_preview");
+```
+
+Lastly, you can choose to specify a resource to generate a token for with
```

Comment:
We should explain why these scoping options are useful — probably for security reasons if they're going to pass the token down to a client??